### ISSUE TITLE
Add support of Home Assistant addon

### ISSUE THREAD

**Issue by T-REX-XP** [binsentsu/am43-ctrl#10]
Hi. Thanks for you contribution, can you add support the Home Assistant  addons structure for this repository ?

https://developers.home-assistant.io/docs/hassio_addon_config/
Thanks

**Comment by binsentsu** [comment]
Hi, isn't this only useful if you would run am43ctrl on the same unit your homeassistant is running on? 
I noticed that the BLE range of these devices is pretty awful, that's why i'm running am43 on a separate rpi close to the blinds. If your willing to contribute this through your fork, feel free to fire a pull request.

**Comment by T-REX-XP** [comment]
> Hi, isn't this only useful if you would run am43ctrl on the same unit your home assistant is running on?

You are right. But I've just wanted to run your project in the easiest way, using docker env. 
There are pros
+  2 clicks install using docker, 
+ integration with HA UI, like the supervisor

>  If your willing to contribute this through your fork, feel free to fire a pull request.

Yesterday I have tried to build addon based on your project but I can't build the docker image based on alpine Linux(by default for Hass addon). There are some problems with node-USB, etc..

> I noticed that the BLE range of these devices is pretty awful, that's why i'm running am43 on a separate rpi close to the blinds

In my opinion, the best way to do it is by using OpemMqttGateway, or something else. 
Pros:
+ more compact
+ not so expensive, (esp32, costs 5$)

Cons:
- MQTT instead of native support
- Currently is not supported by the firmware of the gateway

**Comment by binsentsu** [comment]
Sorry, but I'm not that acquainted with docker stuff. OpenMQTTGateway indeed only supports reading from BLE as far as I can see. In my opinion an ESPHome solution would be the way to go for running on an ESP32. But this requires a rewrite to C++ ...

**Comment by T-REX-XP** [comment]
> Hi. I have found the cool project, that is the custom plugin for ESP Home, 
We can use it for controlling am43 blinds.
Here is the [link](https://github.com/junnikokuki/Meizu-BLE-Thermometer/)
Let's port your code to the c++?))
